ports/sentry-native/portfile.cmake

```
@@ -0,0 +1,39 @@
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://github.com/getsentry/sentry-native/releases/download/0.4.0/sentry-native.zip"
+    FILENAME "sentry-native.zip"
+    SHA512 1ad5e3eb18a85e7fc4e2015c3ba30840173ead19f988f3b85af9081166a889cf0d9f80946f6d8e92a44f58fbe0b86211faa1f1966496c57afd1261637e9b377c
+)
+
+vcpkg_extract_source_archive_ex(
+    OUT_SOURCE_PATH SOURCE_PATH
+    ARCHIVE ${ARCHIVE}
+    NO_REMOVE_ONE_LEVEL
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+)
+
+vcpkg_install_cmake()
+
+vcpkg_copy_pdbs()
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/sentry TARGET_PATH share/${PORT}/cmake)

```

```suggestion
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/sentry TARGET_PATH share/sentry-native/cmake)
```
It should not be updated as `${PORT} `here. Generally, we suggest to use ${PORT} instead of port name in these cases:
license, usage or other files that need to be placed in `share `directory. For CMake files, since the folder name might not be the same as port name.

---

Fixed